### DISCUSSION TITLE
Actions, the danger method doesn't work properly.

### DISCUSSION THREAD

**Discussion by Brand3000** [laravel/nova-issues#5620] [Q&A] [answered]
Hi!
I do 
`PageTranslation::danger('Go', 'Alert')->onlyInline()->showOnDetail()`

I see "Go" as the title but I don't see "Alert" in the body of the popup.

I'm using the last version of Laravel Nova

**Comment by crynobone** [comment, marked as answer]
Shouldn't the usage be:

```php
Action::danger('Go', 'Alert Toast')
    ->confirmText('Alert on the confirmation modal')
    ->onlyInline()
    ->showOnDetail(),
```

**Comment by Brand3000** [comment]
Ok, but where is the "Alert Toast" message" being shown?
